.github/workflows/ci-linux.yaml

```
@@ -11,16 +11,11 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ['ubuntu-latest']
+        os: ['ubuntu-latest','macos-latest']
         perl: 
-          - '5.16'
-          - '5.18'
-          - '5.20'
-          - '5.22'
-          - '5.24'
           - '5.26'
-          - '5.28'
           - '5.30'
+          - 'latest'
```

```suggestion
        os: [ubuntu-latest]
        perl:
          - '5.16'
          - '5.20'
          - '5.26'
          - '5.30'
          - latest
        include:
          - { os: macos-latest, perl: '5.26' }
          - { os: macos-latest, perl: latest }
```
This way it will only test two versions on mac